### PR TITLE
Grammar and style update.

### PR DESCRIPTION
**Pull request by bkmgit**
Minor edit to improve readability.

### CODE REVIEW

File: _episodes/07-find.md [hunk 1]
```diff
@@ -662,11 +662,11 @@ survived so long because it is one of the most productive programming
 environments ever created --- maybe even *the* most productive. Its syntax
 may be cryptic, but people who have mastered it can experiment with
 different commands interactively, then use what they have learned to
-automate their work. Graphical user interfaces may be better at the
-first, but the shell is still unbeaten at the second. And as Alfred
-North Whitehead wrote in 1911, 'Civilization advances by extending the
-number of important operations which we can perform without thinking
-about them.'
+automate their work. Graphical user interfaces may be easier to use at
+first, but the once learned the productivity in the shell is unbeatable.
```
Comment (gdevenyi) [inline _episodes/07-find.md:666]: I think after "once learned" we need a comma.

Comment (gdevenyi) [reply _episodes/07-find.md:666]: transform to "but once learned,"

**Review by gdevenyi — CHANGES_REQUESTED** [review]

**Comment by gdevenyi** [comment]
Thanks for the suggestions, I've made a request for a slight adjustment.

File: _episodes/07-find.md (hunk 1 above)
Comment (bkmgit) [reply _episodes/07-find.md:666]: Thanks

**Comment by gdevenyi** [comment]
Cheers thanks!

**Pull request merged**